lists/eg.csv

```diff
@@ -153,3 +153,5 @@ https://twitter.com/ikhwanweb,FEXP,Free expression and media freedom,2014-04-15,
 https://twitter.com/muhammadmorsi,FEXP,Free expression and media freedom,2014-04-15,citizenlab,
 https://www.facebook.com/ikhwanweb.official,FEXP,Free expression and media freedom,2014-04-15,citizenlab,
 https://www.alaraby.co.uk,FEXP,Free expression and media freedom,2016-10-23,OONI,
+https://whispersystems.org,COMT,Communication Tools,2017-01-19,citizenlab,
+http://textsecure-service-ca.whispersystems.org,COMT,Communication Tools,2017-01-19,citizenlab,Used by Signal desktop,

```

Comment:
I'm actually adding this URL to confirm middlebox blocking - I've anecdotally seen the http version blocked by not https, so I'm not actually expecting a legit response from that server. Adding both to be sure. 

Fixed comma, thanks @jdcc